cnf-tests/testsuites/e2esuite/ovs_qos/ovs_qos.go

```diff
@@ -103,6 +103,7 @@ var _ = Describe("ovs_qos", func() {
 			gSkipReason = "At least two nodes are required for ovs_qos tests."
 			return
 		}
+		fmt.Printf("iperf3BitrateOverride: %s\n", iperf3BitrateOverride)
```

Comment:
I think there was something like ginkgo.Writer to print things

Reply:
changed thanks